libs/sdk-ui-charts/src/highcharts/adapter/HighChartsRenderer.tsx

```diff
@@ -109,6 +104,8 @@ export class HighChartsRenderer extends React.PureComponent<
         this.realignPieOrDonutChart();
     }
 
+    private throttledOnWindowResize = throttle(this.onWindowResize.bind(this), 100);
```

Comment:
maybe we should change onWindowResize to arrow function as well? :)

Reply:
good catch! fixed